packages/vc-api-issuer/src/agent/VcApiIssuer.ts

```diff
@@ -1,7 +1,8 @@
-import {IAgentPlugin, VerifiableCredential} from '@veramo/core'
+import { IAgentPlugin } from '@veramo/core'
 
-import {schema} from '../index'
-import {events, IIssueCredentialArgs, IRequiredContext, IVcApiIssuer, IVcApiIssuerArgs} from '../types/IVcApiIssuer'
+import { schema } from '../index'
+import { events, IIssueCredentialArgs, IRequiredContext, IVcApiIssuer, IVcApiIssuerArgs } from '../types/IVcApiIssuer'
+import { VerifiableCredentialSP } from '@sphereon/ssi-sdk-core'
 
 const fetch = require('cross-fetch')
```

Comment:
mixing node and es6 syntax.

Reply:
Although that was not really part of this PR, moved it to ES6 also for VcApiVerifier